MVVMKit/View Controllers/MVVMCollectionViewController.swift

```diff
@@ -27,31 +27,57 @@ import UIKit
 /**
  A convenience class for a view controller handling a UICollectionView.
  The class fully implements the UICollectionViewDataSource.
+ 
+ `MVVMCollectionViewController` implements the following behaviors:
+ - It sets the data source of the collection view to self.
+ - It implements the viewWillAppear(_:) method and automatically calls `bind(viewModel:)` on first appearance.
+ - It implements the viewDidAppear(_:) method and automatically flashes the collection view's scroll indicators when it first appears.
  */
 open class MVVMCollectionViewController<Model: CollectionViewViewModel>: UIViewController, CollectionViewViewModelOwner, UICollectionViewDataSource {
+    public typealias CustomViewModel = Model
     
     @IBOutlet public weak var collectionView: UICollectionView! {
         didSet { collectionView.dataSource = self }
     }
     
-    public typealias CustomViewModel = Model
-    
-    /// Override this method to bind your view model to the view
-    open func bind(viewModel: Model) {
-        
-    }
-    
     /**
      The view controller view model
      */
     open var viewModel: Model? {
-        didSet { viewModel?.binder = self }
+        didSet {
+            viewModel?.binder = self
+            bindIfViewLoaded()
+        }
     }
     
-    public var sections: [SectionViewModel] {
+    private var sections: [SectionViewModel] {
         return viewModel?.sections ?? []
     }
     
+    /// A boolean value that determines whether the view controller is about to appear the first time.
+    private var needsFirstBind = true
+    
+    // MARK: - Lifecycle
+    
+    open override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        guard needsFirstBind else { return }
+        bind()
+        needsFirstBind = false
+    }
+    
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        collectionView.flashScrollIndicators()
```

Comment:
Don't like this one.
Maybe some subclass doesn't want or need this behaviour.
Consider to make an intermediate superclass inheriting from `MVVMCollectionViewController` (in your app) if you need to share behaviour among multiple subclasses...